src/GraphQL/Types/GraphTypesLookup.cs

```diff
@@ -112,6 +112,18 @@ public void AddType(GraphType type, TypeCollectionContext context)
                         AddTypeIfNotRegistered(arg.Type, context);
                     });
                 }
+
+                if (field.Arguments != null)
+                {
+                    foreach (var arg in field.Arguments)
```

Comment:
Some funkyness with the merge here I think.  The lines above this are already registering the field arguments.